web/themes/joinup/templates/content/node--document--full.html.twig

```diff
@@ -74,7 +74,7 @@
   {{ content.body }}
   {{ content.group_categorisation }}
   {{ content.group_attachment }}
-  <div class="action-link__wrapper right">
+  <div class="fieldset action-link__wrapper right">
```

Comment:
Do we really need all these fieldsets for adding spacing in the case where these are not really fieldset components only?